README.md

```diff
@@ -192,7 +192,7 @@ gosimple: Linter for Go source code that specializes in simplifying a code [fast
 govet (vet, vetshadow): Vet examines Go source code and reports suspicious constructs, such as Printf calls whose arguments do not align with the format string [fast: false, auto-fix: false]
 ineffassign: Detects when assignments to existing variables are not used [fast: true, auto-fix: false]
 staticcheck: Staticcheck is a go vet on steroids, applying a ton of static analysis checks [fast: false, auto-fix: false]
-structcheck: Finds an unused struct fields [fast: true, auto-fix: false]
```

Comment:
I would include this change in another PR

Reply:
Fixed!